### DISCUSSION TITLE
Integrating Autogen with Ollama (on college cluster) to make AI Agents.

### DISCUSSION THREAD

**Discussion by Rushikesh67** [microsoft/autogen#4163] [Q&A]
I plan to create AI agents with AutoGen using the Ollama platform, specifically with the llama3.1:70B model. However, Ollama is hosted on my college’s computer cluster, not on my local computer. I can access the llama models via a URL endpoint (something like https://xyz.com/ollama/api/chat) and an API key provided by the college. Although Ollama has an OpenAI-compatible API, most examples of AutoGen integration involve running Ollama locally, which I can’t do. Is there any way to integrate AutoGen with Ollama using my college's URL endpoint and API key?

**Comment by ekzhu** [comment]
Maybe you can take a look at the current on going PR: https://github.com/microsoft/autogen/pull/4141
